.pylintrc

```diff
@@ -65,7 +65,7 @@ enable=indexing-exception,old-raise-syntax
 # --enable=similarities". If you want to run only the classes checker, but have
 # no Warning level messages displayed, use"--disable=all --enable=classes
 # --disable=W"
-disable=design,similarities,no-self-use,attribute-defined-outside-init,locally-disabled,star-args,pointless-except,bad-option-value,global-statement,fixme,suppressed-message,useless-suppression,locally-enabled,file-ignored,missing-type-doc
+disable=design,similarities,no-self-use,attribute-defined-outside-init,locally-disabled,star-args,pointless-except,bad-option-value,global-statement,fixme,suppressed-message,useless-suppression,locally-enabled,file-ignored,missing-type-doc,missing-return-type-doc
```

Comment:
Add `missing-return-type-doc` to disable. pylint requires `Returns` in docstring to have a type in return aruguments. e.g. 
`Returns:
    List: a list of records`
while in google python style guide the `Returns` should be:
`Returns:
A list of records`